doc/api/fs.md

```diff
@@ -1089,6 +1089,9 @@ For example, the octal value `0o765` means:
 * The group may read and write the file.
 * Others may read and execute the file.
 
+Caveats: on Windows only the write permission can be changed, and there is
```

Comment:
I have not tried all the combinations but I think on Windows effectively you can only change the permission to either 666 or 444.